packages/react/src/ElementFactory/components/EditorElementComponentWrapper.tsx

```diff
@@ -55,8 +50,11 @@ export function EditorElementComponentWrapper<
             />
           ) : undefined
         }
+        // Type cast is needed as getReferenceRect will return `undefined` when
```

Comment:
in that case, we never expect it to be called right? i.e.: getReferenceRect() should only be called when it can return a value?

If that's the case, it's better to throw an Error (as it's an unexpected situation), this will also allow you to change the return type to remove `undefined`